src/stavetie.ts

```diff
@@ -168,7 +160,7 @@ export class StaveTie extends Element {
    * @param firstX specified in pixels
    * @param lastX specified in pixels
    */
-  renderText(firstX: number, lastX: number): void {
+  renderTieText(firstX: number, lastX: number): void {
     if (!this.text) return;
```

Comment:
Is this line still necessary? Can this.text be undefined? I noticed that you changed the constructor to default the text to '' instead of undefined.

Reply:
deleted